From: faqman8 <faqman8@outlook.com>
Date: Sun, 13 Apr 2025 03:37:33 +0000
Subject: Translated using Weblate (Vietnamese)

Currently translated at 22.2% (16 of 72 strings)

Translation: openkylin-nile-new/kylin-os-manager-junk-clean
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kylin-os-manager-junk-clean/vi/
---
 .../kylin-os-manager-device-controller_vi.ts       |  2 +-
 plugins/junk_clean/translations/junk-clean_vi.ts   | 32 +++++++++++-----------
 2 files changed, 17 insertions(+), 17 deletions(-)

diff --git a/plugins/device-controller/translations/kylin-os-manager-device-controller_vi.ts b/plugins/device-controller/translations/kylin-os-manager-device-controller_vi.ts
index b2fe43c..d341c8d 100644
--- a/plugins/device-controller/translations/kylin-os-manager-device-controller_vi.ts
+++ b/plugins/device-controller/translations/kylin-os-manager-device-controller_vi.ts
@@ -809,7 +809,7 @@
         <location filename="../src/include/hwinfostorage.h" line="139"/>
         <location filename="../src/include/hwinfostorage.h" line="153"/>
         <source>Type</source>
-        <translation>Kiểu</translation>
+        <translation>Loại</translation>
     </message>
     <message>
         <location filename="../src/include/hwinfostorage.h" line="56"/>
diff --git a/plugins/junk_clean/translations/junk-clean_vi.ts b/plugins/junk_clean/translations/junk-clean_vi.ts
index b64c68a..c3fd2b3 100644
--- a/plugins/junk_clean/translations/junk-clean_vi.ts
+++ b/plugins/junk_clean/translations/junk-clean_vi.ts
@@ -6,7 +6,7 @@
     <message>
         <location filename="../junk_clean.cpp" line="52"/>
         <source>JunkClean</source>
-        <translation type="unfinished"></translation>
+        <translation>Dọn dẹp rác</translation>
     </message>
 </context>
 <context>
@@ -14,12 +14,12 @@
     <message>
         <location filename="../cleaner/boot_partition_cleaner.cpp" line="50"/>
         <source>boot partition</source>
-        <translation type="unfinished"></translation>
+        <translation>Phân vùng boot</translation>
     </message>
     <message>
         <location filename="../cleaner/boot_partition_cleaner.cpp" line="54"/>
         <source>Boot partition space cleaning</source>
-        <translation type="unfinished"></translation>
+        <translation>Dọn dẹp không gian phân vùng</translation>
     </message>
 </context>
 <context>
@@ -27,27 +27,27 @@
     <message>
         <location filename="../clean_up_entry_widget.cpp" line="86"/>
         <source>%1 in total, clean carefully</source>
-        <translation type="unfinished"></translation>
+        <translation>Tổng cộng%1,Cần thận trọng khi dọn dẹp</translation>
     </message>
     <message>
         <location filename="../clean_up_entry_widget.cpp" line="95"/>
         <source>%1 entries in total, carefully clean up</source>
-        <translation type="unfinished"></translation>
+        <translation>Tổng cộng %1 mục ,cần thận trọng khi dọn dẹp</translation>
     </message>
     <message>
         <location filename="../clean_up_entry_widget.cpp" line="98"/>
         <source>%1 entries</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 mục</translation>
     </message>
     <message>
         <location filename="../clean_up_entry_widget.cpp" line="111"/>
         <source>Scanning...</source>
-        <translation type="unfinished"></translation>
+        <translation>Đang quét...</translation>
     </message>
     <message>
         <location filename="../clean_up_entry_widget.cpp" line="115"/>
         <source>Very clean</source>
-        <translation type="unfinished"></translation>
+        <translation>Rất sạch sẽ</translation>
     </message>
 </context>
 <context>
@@ -55,27 +55,27 @@
     <message>
         <location filename="../clean_up_finish_widget.cpp" line="66"/>
         <source>Complete</source>
-        <translation type="unfinished"></translation>
+        <translation>Hoàn tất</translation>
     </message>
     <message>
         <location filename="../clean_up_finish_widget.cpp" line="76"/>
         <source>Deep clean</source>
-        <translation type="unfinished"></translation>
+        <translation>Dọn dẹp sâu</translation>
     </message>
     <message>
         <location filename="../clean_up_finish_widget.cpp" line="107"/>
         <source>Successfully cleaned up %1 of garbage</source>
-        <translation type="unfinished"></translation>
+        <translation>Đã dọn dẹp thành công %1 rác</translation>
     </message>
     <message>
         <location filename="../clean_up_finish_widget.cpp" line="111"/>
         <source>Cleaning files: %1 entries</source>
-        <translation type="unfinished"></translation>
+        <translation>Dọn dẹp tệp: %1 tệp</translation>
     </message>
     <message>
         <location filename="../clean_up_finish_widget.cpp" line="115"/>
         <source>Cleaning time: %1 second</source>
-        <translation type="unfinished"></translation>
+        <translation>Thời gian dọn dẹp: %1 giây</translation>
     </message>
 </context>
 <context>
@@ -83,17 +83,17 @@
     <message>
         <location filename="../clean_up_group_widget.cpp" line="61"/>
         <source>%1 items of %2 in total, %3 selected</source>
-        <translation type="unfinished"></translation>
+        <translation>Tổng cộng %1 mục %2,đã chọn %2</translation>
     </message>
     <message>
         <location filename="../clean_up_group_widget.cpp" line="67"/>
         <source>%1 items with %2 traces, %3 selected</source>
-        <translation type="unfinished"></translation>
+        <translation>Tổng cộng %1 mục %2 dấu vết, đã chọn %3 dấu vết</translation>
     </message>
     <message>
         <location filename="../clean_up_group_widget.cpp" line="91"/>
         <source>System junk</source>
-        <translation type="unfinished"></translation>
+        <translation>Rác hệ thống</translation>
     </message>
     <message>
         <location filename="../clean_up_group_widget.cpp" line="93"/>
